ttnn/cpp/ttnn/operations/data_movement/clone/device/clone_device_operation.cpp

```diff
@@ -0,0 +1,64 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "clone_device_operation.hpp"
+
+namespace ttnn::operations::data_movement::clone {
+void CloneOperation::validate_inputs(
+    const operation_attributes_t& operation_attributes, const tensor_args_t& tensor_args) {
```

Comment:
can we prefix all of these FATAL messages with ttnn.clone?

Reply:
When a FATAL error occurs, the call stack shows that the issue originates from `ttnn::clone`, and adding a prefix to the message could become inconvenient when reusing code in other operations. Is the prefix absolutely necessary?

Reply:
I agree with @hschoi4448. 
However, I still made some adjustments; the FATAL message has been changed to 'Clone: ...'

Reply:
For code reuse, the prefixes can be handy for forcing you to re-evaluate your assumptions as you reuse code in other ops. Thanks for making the change :)